platform/platform-sample-package.md

```diff
@@ -25,7 +25,7 @@ snippet: Launch
 The Platform Sample package performs the following when the Console App is started:
 
 1. Copies platform binaries into the project output directory.
-1. Finds available ports for all the platform tools.
+1. Find available ports for all the platform tools.
```

Comment:
I think "Finds" is correct here gramatically.

Reply:
Good catch, fixed